internal/allocator/allocator.go

```diff
@@ -6,6 +6,7 @@ import (
 	"math"
 	"net"
 
+	"github.com/mikioh/ipaddr"
```

Comment:
nit, I think the order is: std lib, empty line, our libs, empty line, 3rd party

Reply:
Will do!